frontend/lib/src/components/core/LibContext.tsx

```diff
@@ -58,6 +70,15 @@ export interface LibContextProps {
    * will default to false for regular streamlit
    */
   hideFullScreenButtons: boolean
+
+  /** The configuration from the apps host. */
+  hostConfig: HostConfig
+
+  /**
+   * Set the configuration from the app's host.
+   * @see App.setAndSendTheme
```

Comment:
Double checking: Is this documentation correct?

Reply:
ups, yep. That needs to be changed to:

```suggestion
   * @see App.setHostConfig
```